sdk/core/core-http/lib/atomXmlOperationSpec.ts

```diff
@@ -0,0 +1,8 @@
+import { AtomXmlSerializer } from "./atomXmlSerializer";
```

Comment:
Missing copyright

Reply:
Addressed this and all other comments below with code changes @chradek @ramya-rao-a